upgrade-instructions.md

```diff
@@ -37,6 +37,10 @@ In order to enforce task sequential processing at the cluster level, we rely on
 
 Users of distributed James product thus need to upgrade their RabbitMQ server to be at least version 3.8.
 
+To perform the upgrade the RabbitMQ queue `taskManagerWorkQueue` have to be deleted. James will then recreate it with the single active consumer property.
+In order not to lose any messages, all the messages should have been processed, thus this queue should be empty.
```

Comment:
s/not to lose/to not loose?

Reply:
No, here we want the verb : `to lose / lost / lost`
not the adjective `Loose` meaning untight

Reply:
ah yeah, my bad :)